### PR TITLE
PLANET-7165 Re-write Covers campaign test in Playwright

### PR DESCRIPTION
**Pull request by sagarsdeshmukh**
Ref: https://jira.greenpeace.org/browse/PLANET-7165

- Added test for Covers campaign block
- Removed old codeception test

**Testing:**
- Run ` npx playwright test tests/e2e/covers-campaign.spec.js` on branch locally.

### CODE REVIEW

**Comment by planet-4** [comment]
### Test instance is ready :rocket:

:new_moon: [telesto](https://www-dev.greenpeace.org/test-telesto) | [admin](https://www-dev.greenpeace.org/test-telesto/wp-admin/) | [blocks report](https://www-dev.greenpeace.org/test-telesto/wp-admin/?page=plugin_blocks_report) | [CircleCI](https://app.circleci.com/pipelines/github/greenpeace/planet4-test-telesto) | [composer-local.json](https://github.com/greenpeace/planet4-test-telesto/blob/main/composer-local.json)

:watch: 2023.06.09 11:28:25

File: tests/e2e/tools/lib/covers.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+const {expect} = require('@playwright/test');
+
+const TAG_NAMES = ['Climate', 'Oceans','renewables', 'Consumption'];
+
+async function addCoversBlock(page, style) {
+  // Add Covers block.
+  await page.locator('.block-editor-block-list__layout').click();
+  await page.locator('p.is-selected.wp-block-paragraph').fill('/planet-4-covers');
```
[the remark this answers is not in the document]
Comment (sagarsdeshmukh) [reply tests/e2e/tools/lib/covers.js]: PR updated!

[the remark this answers is not in the document]
Comment (Osong-Michael) [reply tests/e2e/tools/lib/covers.js]: Hey @sagarsdeshmukh, one more thing, please can your rebase with the main branch, thanks

[the remark this answers is not in the document]
Comment (sagarsdeshmukh) [reply tests/e2e/tools/lib/covers.js]: PR rebased with main branch 🙂

**Review by Osong-Michael — APPROVED** [review]

**Pull request merged**